--- a/src/Commands/ModuleMakeFilamentThemeCommand.php
+++ b/src/Commands/ModuleMakeFilamentThemeCommand.php
@@ -5,6 +5,7 @@ namespace Coolsam\Modules\Commands;
 use Coolsam\Modules\Facades\FilamentModules;
 use Filament\Commands\MakeThemeCommand;
 use Filament\Panel;
+use Illuminate\Filesystem\Filesystem;
 use Nwidart\Modules\Module;

 use function Laravel\Prompts\text;
@@ -15,8 +16,10 @@ class ModuleMakeFilamentThemeCommand extends MakeThemeCommand

     protected $description = 'Create a new Filament theme in a module';

-    public function handle(): int
+    public function handle(Filesystem $filesystem): int
     {
+        $this->filesystem = $filesystem;
+
         $module = $this->getModule();

         $this->call('vendor:publish', [
